go/worker/client/committee/node.go

```diff
@@ -140,60 +139,28 @@ func (n *Node) Query(ctx context.Context, round uint64, method string, args []by
 	// Fetch the active descriptor so we can get the current message limits.
 	n.commonNode.CrossNode.Lock()
 	dsc := n.commonNode.CurrentDescriptor
-	latestRound := n.commonNode.CurrentBlock.Header.Round
 	n.commonNode.CrossNode.Unlock()
 
 	if dsc == nil {
 		return nil, api.ErrNoHostedRuntime
 	}
 	maxMessages := dsc.Executor.MaxMessages
 
-	var resolvedRound uint64
```

Comment:
Should this be feature-based? Otherwise things will degrade immediately after consensus/node upgrade until the runtime is also upgraded.

Reply:
Yes, as the sent block would be one height lower.